Common/API/LiveAlgorithmSettings.cs

```diff
@@ -0,0 +1,299 @@
+            Password = "";
+            Environment = environment;
+            Account = account;
+            Id = brokerage.ToString();
+        }
+        /// <summary>
+        /// 'Interactive' / 'FXCM' / 'Oanda' / 'Tradier' /'PaperTrading'
+        /// </summary>
+        [JsonProperty(PropertyName = "id")]
+        public string Id { get; private set; }
+
+        /// <summary>
+        /// Username associated with brokerage
+        /// </summary>
+        [JsonProperty(PropertyName = "user")]
+        public string User { get; private set; }
+
+        /// <summary>
+        /// Password associated with brokerage
+        /// </summary>
+        [JsonProperty(PropertyName = "password")]
+        public string Password { get; private set; }
+
+        /// <summary>
+        /// 'live'/'paper'
+        /// </summary>
+        [JsonProperty(PropertyName = "environment")]
+        public string Environment { get; private set; }
+
+        /// <summary>
+        /// Account of the associated brokerage
+        /// </summary>
+        [JsonProperty(PropertyName = "account")]
+        public string Account { get; private set; }
+    }
+
+    /// <summary>
+    /// Default live algorithm settings
+    /// </summary>
+    public class DefaultLiveAlogrithmSettings : BaseLiveAlgorithmSettings
+    {
+        /// <summary>
+        /// Constructor for default algorithms
+        /// </summary>
+        /// <param name="user">Username associated with brokerage</param>
+        /// <param name="password">Password associated with brokerage</param>
+        /// <param name="brokerage">Name of the brokerage</param>
+        /// <param name="environment">'live'/'paper'</param>
+        /// <param name="account">Account id for brokerage</param>
+        public DefaultLiveAlogrithmSettings(string user,
+                                                string password,
+                                                BrokerageName brokerage,
+                                                string environment,
+                                                string account)
+            : base(user, password, brokerage, environment, account)
+        {
+        }
+    }
+
+    /// <summary>
+    /// Algorithm setting for trading with FXCM
+    /// </summary>
+    public class FXCMLiveAlogrithmSettings : BaseLiveAlgorithmSettings
+    {
+
+        /// <summary>
+        /// Contructor for live trading with FXCM
+        /// </summary>
+        /// <param name="user">Username associated with brokerage</param>
+        /// <param name="password">Password associated with brokerage</param>
+        /// <param name="brokerage">Name of the brokerage</param>
+        /// <param name="environment">'live'/'paper'</param>
+        /// <param name="account">Account id for brokerage</param>
+        public FXCMLiveAlogrithmSettings(string user,
+                                         string password,
+                                         BrokerageName brokerage,
```

Comment:
the "FXCMLiveAlogrithmSettings" can't accept any other brokerage names; this shouldn't be a constructor arg. Same for all the other *LiveAlogrithmSettings parameters
```base(user, password, BrokerageName.FxcmBrokerage, environment, account)"```

Tradier cannot do "paper" that environment can be removed there, Tradier "string lifetime," is always 86399 seconds